WinUIGallery/Samples/ControlPages/RatingControlPage.xaml

```diff
@@ -27,11 +27,11 @@
             <controls:ControlExample.Options>
                 <StackPanel Width="220">
                     <CheckBox x:Name="clearEnabledCheck" Content="IsClearEnabled" />
-                    <TextBlock Text="Swipe left to clear your rating." TextWrapping="WrapWholeWords" />
+                    <TextBlock Text="Click again to clear your rating." TextWrapping="WrapWholeWords" />
```

Comment:
Hold on? I think _both_ of these gestures are valid.

Reply:
I know, but I think most of the users here are using the mouse.

Reply:
Sweeping can be done with touch and mouse

Reply:
Agree with @Jay-o-Way here

Reply:
"Swipe left or click again to clear your rating."?